app/decorators/event_decorator.rb

```diff
@@ -0,0 +1,13 @@
+# :reek:NilCheck
+
+class EventDecorator < Draper::Decorator
+  delegate_all
+
+  def already_subscribed_by_user?(user_id)
+    attendances.find_by(user_id: user_id)&.id
+  end
+
+  def editable?(user_id)
+    organization.users.find_by(id: user_id).present?
```

Comment:
not clear why lookup for user is in event decorator, it seems we've got some misunderstanding

Reply:
@anatoliliotych  i need to check if current user can edit particular event. And when this verify was in user decorate, it was also strange